src/Symfony/Bundle/FrameworkBundle/Command/WorkerAmqpMoveCommand.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class WorkerAmqpMoveCommand extends ContainerAwareCommand
+{
+    private $broker;
+    private $logger;
+
+    protected function configure()
+    {
+        $this
+            ->setName('worker:amqp:move')
+            ->setDescription('Take all messages from a queue, and send them to the default exchange with a new routing key.')
+            ->setDefinition(array(
+                new InputArgument('from', InputArgument::REQUIRED, 'The queue.'),
+                new InputArgument('to', InputArgument::REQUIRED, 'The new routing key.'),
+            ))
+        ;
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        $this->broker = $this->getContainer()->get('amqp.broker');
+        $this->logger = $this->getContainer()->get('logger');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $from = $input->getArgument('from');
+        $to = $input->getArgument('to');
+
+        while (false !== $message = $this->broker->get($from)) {
+            $this->logger->info('Move a message...');
+            $this->broker->move($message, $to);
+            $this->broker->ack($message);
+            $this->logger->debug('...message moved.');
```

Comment:
Why not use the same level? Especially since there are ellipsis that should correspond to each other?